### DISCUSSION TITLE
How to display my gists?

### DISCUSSION THREAD

**Discussion by Coder-ACJHP** [community/community#142442] [Mobile]
Hi, i would like to access my gists from the mobile application but i couldn't find, is Github mobile supports viewing gists?
Thanks in advance.

**Comment by fxrdhan** [comment]
Hi! You're right - currently, the GitHub mobile app doesn't have direct support for viewing or managing Gists. Here are your options to access your Gists:

1. **Mobile Browser Method**
   - Go to `gist.github.com` in your mobile browser
   - Sign in to your GitHub account
   - You'll have full access to view, create, and edit your Gists

2. **Alternative Methods**:
   - Use the desktop version of GitHub
   - Access through third-party GitHub apps that support Gists
   - Bookmark `gist.github.com` on your mobile browser for quick access
